django_project/certification/tests/tests.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
```

Comment:
hi @ann26 
maybe you could create test file based on model instead put all of model test in one file.
for example tests_certifiying_organisation.py just for class TestCertifyingOrganisation